app/javascript/components/action-form/action-form.schema.js

```diff
@@ -193,12 +193,16 @@ function createSchema(recordId, promise, inheritTags, evaluateAlert, tags, ansib
               name: 'attribute',
               id: 'attribute',
               label: 'attribute',
+              isRequired: true,
+              validate: [{ type: validatorTypes.REQUIRED }],
```

Comment:
Does this force them to enter an attribute pair?

Reply:
Easier to just show how it works but yes it wont require the pairs unless you tell it you want to include a pair

https://user-images.githubusercontent.com/29209973/184922530-f1a24bbf-8c04-4015-abbb-82a4eda8a54f.mov

Reply:
Oh perfect.